server/routes/specialRemissionRoutes.ts

```diff
@@ -162,6 +164,13 @@ export default class SpecialRemissionRoutes {
     const { nomsId } = req.params
 
     const adjustment = this.adjustmentsStoreService.getOnly(req, nomsId)
+
+    const crdString = await this.calculateReleaseDatesService.getCrdForSpecialRemission(nomsId, username)
```

Comment:
Might want to add a try catch rather than ` crdString ? `

```
let fromDate = new Date()
try {
    const crdString = await this.calculateReleaseDatesService.getCrdForSpecialRemission(nomsId, username).         
    const crdDate = new Date(crdString)
    fromDate = new Date(crdDate.setDate(crdDate.getDate() - adjustment.days))
} catch() {}
```

Reply:
I've updated to remove the ? new Date() doesn't throw an error with an empty string